<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbsorptionCorrections</class>
 <widget class="QWidget" name="AbsorptionCorrections">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Absoprtion Corrections</string>
  </property>
  <layout class="QVBoxLayout" name="loAbsorptionCorrections">
   <item>
    <widget class="QGroupBox" name="gbInput">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="1" column="0">
       <widget class="QCheckBox" name="ckUseCan">
        <property name="text">
         <string>Use Container:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="MantidQt::MantidWidgets::DataSelector" name="dsSampleInput" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoLoad" stdset="0">
         <bool>true</bool>
        </property>
        <property name="workspaceSuffixes" stdset="0">
         <stringlist>
          <string>_red</string>
          <string>_sqw</string>
         </stringlist>
        </property>
        <property name="fileBrowserSuffixes" stdset="0">
         <stringlist>
          <string>_red.nxs</string>
          <string>_sqw.nxs</string>
         </stringlist>
        </property>
        <property name="showLoad" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbSampleInput">
        <property name="text">
         <string>Sample Input:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="MantidQt::MantidWidgets::DataSelector" name="dsCanInput" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoLoad" stdset="0">
         <bool>true</bool>
        </property>
        <property name="workspaceSuffixes" stdset="0">
         <stringlist>
          <string>_red</string>
          <string>_sqw</string>
         </stringlist>
        </property>
        <property name="fileBrowserSuffixes" stdset="0">
         <stringlist>
          <string>_red.nxs</string>
          <string>_sqw.nxs</string>
         </stringlist>
        </property>
        <property name="showLoad" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbShapeDetails">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Shape Details</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="loShape">
        <item>
         <widget class="QLabel" name="lbShape">
          <property name="text">
           <string>Shape:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbShape">
          <item>
           <property name="text">
            <string>Flat Plate</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Annulus</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cylinder</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="swShapeDetails">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pgAbsCorFlatPlate">
         <layout class="QGridLayout" name="loFlatPlate">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="3" column="0">
           <widget class="QLabel" name="lbFlatCanFrontThickness">
            <property name="text">
             <string>Container Front Thickness:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbFlatSampleWidth">
            <property name="text">
             <string>Sample Width:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDoubleSpinBox" name="spFlatElementSize">
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="spFlatCanFrontThickness">
            <property name="suffix">
             <string> cm</string>
            </property>
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="spFlatSampleHeight">
            <property name="suffix">
             <string> cm</string>
            </property>
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="lbFlatElementSize">
            <property name="text">
             <string>Element Size:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spFlatSampleWidth">
            <property name="suffix">
             <string> cm</string>
            </property>
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spFlatSampleThickness">
            <property name="suffix">
             <string> cm</string>
            </property>
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lbFlatSampleThickness">
            <property name="text">
             <string>Sample Thickness:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="lbFlatSampleHeight">
            <property name="text">
             <string>Sample Height:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QDoubleSpinBox" name="spFlatCanBackThickness">
            <property name="suffix">
             <string> cm</string>
            </property>
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="lbFlatCanBackThickness">
            <property name="text">
             <string>Container Back Thickness:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pgAbsCorAnnulus">
         <layout class="QGridLayout" name="loAnnulus">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spAnnSampleInnerRadius">
            <property name="suffix">
             <string> cm</string>
            </property>
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="lbAnnSampleOuterRadius">
            <property name="text">
             <string>Sample Outer Radius:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spAnnEvents">
            <property name="maximum">
             <number>1000000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>5000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbAnnSampleInnerRadius">
            <property name="text">
             <string>Sample Inner Radius:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lbCylEvents">
            <property name="text">
             <string>Neutron Events:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="spAnnSampleOuterRadius">
            <property name="suffix">
             <string> cm</string>
            </property>
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QDoubleSpinBox" name="spAnnCanOuterRadius">
            <property name="suffix">
             <string> cm</string>
            </property>
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="lbAnnCanOuterRadius">
            <property name="text">
             <string>Container Outer Radius:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="spAnnCanInnerRadius">
            <property name="suffix">
             <string> cm</string>
            </property>
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lbAnnCanInnerRadius">
            <property name="text">
             <string>Container Inner Radius:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pgAbsCorCylinder">
         <layout class="QGridLayout" name="loCylinder">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lbCylSampleRadius">
            <property name="text">
             <string>Sample Radius:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spCylEvents">
            <property name="maximum">
             <number>1000000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>5000</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lbAnnEvents">
            <property name="text">
             <string>Neutron Events:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="lbCylCanRadius">
            <property name="text">
             <string>Container Radius:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spCylSampleRadius">
            <property name="suffix">
             <string> cm</string>
            </property>
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDoubleSpinBox" name="spCylCanRadius">
            <property name="suffix">
             <string> cm</string>
            </property>
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbSampleDetails">
     <property name="title">
      <string>Sample Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="spSampleNumberDensity">
        <property name="suffix">
         <string> A^-3</string>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lbSampleChemicalFormula">
        <property name="text">
         <string>Chemical Formula:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbSampleNumberDensity">
        <property name="text">
         <string>Number Density: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="leSampleChemicalFormula">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbContainerDetails">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Container Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="2">
       <widget class="QLabel" name="lbCanChemicalFormula">
        <property name="text">
         <string>Chemical Formula:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbCanNumberDensity">
        <property name="text">
         <string>Number Density:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="spCanNumberDensity">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> A^-3</string>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="leCanChemicalFormula">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="spCanScale">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="ckScaleCan">
        <property name="text">
         <string>Scale:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="ckUseCanCorrections">
        <property name="text">
         <string>Use Container Corrections</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOutput">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Output Options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="ckKeepFactors">
        <property name="text">
         <string>Keep Correction Factors</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="ckPlot">
        <property name="text">
         <string>Plot Result</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="ckSave">
        <property name="text">
         <string>Save Result</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::DataSelector</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/DataSelector.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ckUseCan</tabstop>
  <tabstop>cbShape</tabstop>
  <tabstop>spFlatSampleWidth</tabstop>
  <tabstop>spFlatSampleHeight</tabstop>
  <tabstop>spFlatSampleThickness</tabstop>
  <tabstop>spFlatCanFrontThickness</tabstop>
  <tabstop>spFlatCanBackThickness</tabstop>
  <tabstop>spFlatElementSize</tabstop>
  <tabstop>spAnnSampleInnerRadius</tabstop>
  <tabstop>spAnnSampleOuterRadius</tabstop>
  <tabstop>spAnnCanInnerRadius</tabstop>
  <tabstop>spAnnCanOuterRadius</tabstop>
  <tabstop>spAnnEvents</tabstop>
  <tabstop>spCylSampleRadius</tabstop>
  <tabstop>spCylCanRadius</tabstop>
  <tabstop>spCylEvents</tabstop>
  <tabstop>spSampleNumberDensity</tabstop>
  <tabstop>leSampleChemicalFormula</tabstop>
  <tabstop>ckUseCanCorrections</tabstop>
  <tabstop>ckScaleCan</tabstop>
  <tabstop>spCanScale</tabstop>
  <tabstop>spCanNumberDensity</tabstop>
  <tabstop>leCanChemicalFormula</tabstop>
  <tabstop>ckKeepFactors</tabstop>
  <tabstop>ckPlot</tabstop>
  <tabstop>ckSave</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cbShape</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>swShapeDetails</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ckScaleCan</sender>
   <signal>toggled(bool)</signal>
   <receiver>spCanScale</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ckUseCan</sender>
   <signal>toggled(bool)</signal>
   <receiver>dsCanInput</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ckUseCan</sender>
   <signal>toggled(bool)</signal>
   <receiver>gbContainerDetails</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ckUseCanCorrections</sender>
   <signal>toggled(bool)</signal>
   <receiver>leCanChemicalFormula</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>528</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ckUseCanCorrections</sender>
   <signal>toggled(bool)</signal>
   <receiver>spCanNumberDensity</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spFlatCanFrontThickness</sender>
   <signal>valueChanged(double)</signal>
   <receiver>spFlatCanBackThickness</receiver>
   <slot>setValue(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
